docs/micro_frontends/overriding_brand_specific_elements.rst

```diff
@@ -0,0 +1,22 @@
+**********************************
+Overriding Brand Specific Elements
+**********************************
+
+MFEs contain Open edX branded headers, footers and style. To build a MFE to reflect the brand of the particular Open edX instance some npm dependencies are designed to be overridden. **Overriding packages must expose the same interface as the dependencies they are overriding**.
+
+Example: for edx.org the Open edX @edx/frontend-component-header is overridden with @edx/frontend-component-header-edx using npm aliases (`introduced in npm version 6.9.0 <https://github.com/npm/rfcs/blob/latest/implemented/0001-package-aliases.md>`_). Example syntax below:
+
+.. code-block:: bash
+
+  # Syntax: npm install <package-name>@<type>:<branded-package>
+
+  # npm package
+  npm install @edx/frontend-component-header@npm:@edx/frontend-component-header-edx@latest
+
+  # git repository
+  npm install @edx/frontend-component-header@git:https://github.com/edx/frontend-component-header-edx.git
+
```

Comment:
I know this was wrriten before, but I guess this can good opportunity to change it. 

The above example of installing using `git` is incorrect, it can be one of these: 

`npm install @edx/frontend-component-header@git+https://github.com/edx/frontend-component-header-edx.git`
Or
`npm install @edx/frontend-component-header@git://github.com/edx/frontend-component-header-edx.git`
ref: https://docs.npmjs.com/cli/v8/commands/npm-install